www/docs/react/react-intro.md

```diff
@@ -0,0 +1,85 @@
+---
+id: intro
+title: Usage with React
+sidebar_label: Usage with React
+slug: /react
+---
+
+> If you're using Next.js, read the [Usage with Next.js](/docs/nextjs) guide instead.
+
+## Install tRPC dependencies
+
+```bash
+yarn add @trpc/client @trpc/server @trpc/react react-query zod
+```
+
+- React Query: @trpc/react provides a thin wrapper over [react-query](https://react-query.tanstack.com/overview). It is required as a peer dependency.
+- Zod: most examples use Zod for input validation, though it isn't required. You can use a validation library of your choice (Yup, io-ts, etc). In fact, any object containing a `parse` or `validateSync` method will work.
+
+## Implement your `appRouter`
+
+Follow the [Quickstart](/docs/quickstart) and read the [`@trpc/server` docs](/docs/procedures) for guidance on this. Once you have your API implemented and listening via HTTP, continue to the next step.
+
+## Create tRPC hooks
+
+Create a set of strongly-typed React hooks from your `AppRouter` type signature with `createReactQueryHooks`.
+
+```tsx
+// utils/trpc.ts
+import { createReactQueryHooks } from '@trpc/react';
+import type { AppRouter } from '../server/trpc';
```

Comment:
you refer to `../server` here but in other place you refer to `../trpc` - pick one

Reply:
I used different file names in the plain React examples (where you're probably importing the type from an Express server elsewhere on the file system) and the Next.js examples (where it's being imported from `pages/api/trpc/[trpc].ts`. I'll make it more clear here that you should import from wherever your router is implemented.